### PR TITLE
added docker usage guide and added explanation for #437

### PR DESCRIPTION
**Pull request by temmey**
## Basics

- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [x] The PR is rebased with current master.

## Checklist

- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great job, thanks for your first PR! I invited you in the team.

**Comment by markus2330** [comment]
Something went wrong here, please rebase properly or create a new branch with your commits on top of it (cherry-pick). You can ask e.g. @badnames how to do this.

File: doc/backend/01setup.md [hunk 1]
```diff
@@ -13,10 +13,14 @@ All steps mentioned here have to be executed in the `backend/` folder.
...
 - `RUST_LOG` used to set the logging config for [env_logger](https://docs.rs/env_logger/latest/env_logger/)
 
+The user named 'permaplant' doesn't have the required permissions to start the 'postgis' extension. You'll have to set it up yourself using the postgres user.
+
 Ensure that you grant the necessary permissions for the user to use Postgres. One way to do this is by using the following command:
 
+
 ```shell
 sudo -u postgres psql
+CREATE EXTENSION IF NOT EXISTS postgis;
```
[the remark this answers is not in the document]
Comment (temmey) [reply doc/backend/01setup.md:25]: fixed in [4501f86](https://github.com/ElektraInitiative/PermaplanT/pull/460/commits/4501f86e91b4a64444277d71acd885ce3522d4e0)

File: doc/backend/README.md [hunk 2]
```diff
@@ -8,6 +8,28 @@
 - libssl-dev
 - pkg-config
 
+
+Instead of setting up psql server on your local machine, you could use a 'postgis' Docker container as another option.
```
[the remark this answers is not in the document]
Comment (temmey) [reply doc/backend/README.md]: fixed in [0716c39](https://github.com/ElektraInitiative/PermaplanT/pull/460/commits/0716c39c6729cbea51845030365daa9ae6b7775f)

File: doc/backend/01setup.md [hunk 3]
```diff
@@ -13,10 +13,14 @@ All steps mentioned here have to be executed in the `backend/` folder.
 - `AUTH_CLIENT_ID` the client id the frontend should use to log in
 - `RUST_LOG` used to set the logging config for [env_logger](https://docs.rs/env_logger/latest/env_logger/)
 
+The user named 'permaplant' doesn't have the required permissions to start the 'postgis' extension. You'll have to set it up yourself using the postgres user.
```
[the remark this answers is not in the document]
Comment (temmey) [reply doc/backend/01setup.md]: fixed in [4501f86](https://github.com/ElektraInitiative/PermaplanT/pull/460/commits/4501f86e91b4a64444277d71acd885ce3522d4e0)

**Comment by markus2330** [comment]
There is still something wrong in the rebase, please verify that https://github.com/ElektraInitiative/PermaplanT/pull/460/files contains exactly and exclusively the changes you want. You can also reopen the branch/PR if you don't know what happened.

There are some build server issues, too. See doc/ci.md

**Comment by markus2330** [comment]
jenkins build please

**Comment by markus2330** [comment]
I assume #503 is the follow up. Please reopen if needed.

**Pull request closed without merging**